src/Providers/AppServiceProvider.php

```diff
@@ -160,6 +160,7 @@ public function register()
     protected function registerMiddlewareGroup()
     {
         $this->app->make(Router::class)->middlewareGroup('statamic.web', [
+            \Statamic\Http\Middleware\AddQueuedCookiesToResponse::class,
```

Comment:
Why is this needed? It's already on the web middleware by default. https://github.com/laravel/laravel/blob/e87bfd60ed4ca30109aa73c4d23250c113fe75ff/app/Http/Kernel.php#L34

Reply:
Yeah sorry I had meant to remove it. Initially thought it was needed but later realised it was to do with needing to queue cookies. Thanks for highlighting it.

Reply:
Nice, thanks.